data/human/news.txt

```diff
@@ -289,6 +289,9 @@ news "unemployed youth"
 			"Do you know how lucky you are, that you can visit a different world every week if you want to? I've been stuck on this one filthy little planet my whole life."
 			"If I had enough money, I'd go up to a captain in the spaceport and ask for transport to anywhere else. Nowhere's as bad as this place."
 			"Are you hiring new crewmembers? And are you by chance planning to fly to the Paradise Worlds?"
+			"Whatcha looking at, ya old good-fur-nothing?"
```

Comment:
To be honest, this feels like something a sour old man would say to a kid, rather than the reverse (minus the word "old" of course). :)

Reply:
Perhaps something more like, "Whatcha looking at, ya old geezer?" or "old fart" or "old fossil?"

Reply:
There ya go, much better!